docs/adrs/0017-laptop-security.md

```diff
@@ -0,0 +1,22 @@
+# ADR-0017 Laptop Security
+
+## Context
+
+When people leave their laptop open and unlocked this could be a big security breach, especially when you have SSO on
+for multiple services.
+
+## Decision
+
+Whenever a colleague will find your laptop open and unlocked, they will message in the `random` Mattermost channel of 
+the Rijks AI Validation team to bring cookies and cake to the office the next thursday as a way of compensation.
```

Comment:
```suggestion
the Rijks AI Validation team to bring cookies and/or cake to the office the next thursday as a way of compensation.
```

Reply:
```suggestion
the Rijks AI Validation team to bring cookies or cake to the office the next thursday as a way of compensation.
```
I think 'or' is logically sufficient. Consider maybe not specifying the specific compensation, now it is limited to cookies or cake. Maybe create a more abstract category of compensation that includes, but does not limit us to, cookies or cake.